src/main/java/uk/gov/companieshouse/api/accounts/model/validation/Errors.java

```diff
@@ -13,7 +13,7 @@
 
     @JsonProperty("errors")
     @Field("errors")
-    private Set<Error> errors = new HashSet<>();
+    private Set<Error> errorsList = new HashSet<>();
```

Comment:
Same as the previous comment, could this cause issues with spring?

Reply:
I don't think so. See previous answer.

Reply:
errorSet?